forte/data/ontology/top.py

```diff
@@ -686,5 +688,169 @@ def get(
         yield from self.pack.get(entry_type, self, components, include_sub_type)
 
 
+class ImageAnnotation(Entry):
+    def __init__(self, pack: PackType, image_payload_idx: int):
+        """
+        ImageAnnotation type entries, such as "edge" and "bounding box".
+        Each ImageAnnotation has a ``image_payload_idx`` corresponding to its
+        image representation in the payload array.
+
+        Args:
+            pack: The container that this image annotation
+                will be added to.
+            image_payload_idx: A integer that represents the index of
+                the image in the payloads.
+        """
+        self._image_payload_idx = image_payload_idx
+        super().__init__(pack)
```

Comment:
I think normally we want to do `__init__` first unless necessary.

Reply:
moved `__init__` up.